packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -27,14 +29,10 @@ export interface SnapWPEnv {
 	 * REST URL prefix. Defaults to `/wp-json`.
 	 */
 	restUrlPrefix: string;
-	/**
-	 * URL prefix for WP assets loaded from 'wp-includes' dir . Defaults to `/proxy`.
-	 */
-	corsProxyPrefix: string;
 	/**
 	 * Flag to enable cors middleware which proxies assets from WP server.
 	 */
-	useCorsProxy?: boolean;
+	hasCorsProxy?: string | false;
```

Comment:
Must not have been clear in my previous comment.

The _exported_ config should have a `corsProxyPrefix?: string | undefined` _and_ a `hasCorsProxy:bool`.

The difference is that that `hasCorsProxy = !! corsProxyPrefix` instead of needing to be defined/managed by a separate ENV var.